dts/arm/nxp/nxp_k6x.dtsi

```diff
@@ -393,6 +393,17 @@
 			status = "disabled";
 			label = "USBD";
 		};
+
+		eth: ethernet@400C0004 {
+			compatible = "nxp,mcux-ethernet";
+			reg = <0x400C0004 0x620>;
+			interrupts = <42 0>;
+			status = "disabled";
+			local-mac-address = [00 11 22 33 44 55];
```

Comment:
We should probably have this be:
`local-mac-address = [00 00 00 00 00 00];`

and let the board dts file set it do a different value.